File: miniJobs/Application/Jobs/Models/JobStep1Request.cs
namespace Application.Jobs.Models;

public class JobStep1Request
{
    public int? Id { get; set; }
    public string Name { get; set; }
    public string StreetAddressAndNumber { get; set; }
    public string Description { get; set; }
    public int CityId { get; set; }
}

File: miniJobs/UI/minijobs_mobile/lib/models/job/job_step1_request.dart
import 'package:json_annotation/json_annotation.dart';

part 'job_step1_request.g.dart';

@JsonSerializable()
class JobStep1Request {
  String? name;
  String? description;
  String? streetAddressAndNumber;
  int? cityId;


  JobStep1Request(this.name, this.description, this.streetAddressAndNumber, this.cityId);
  Map<String, dynamic> toJson() => _$JobStep1RequestToJson(this);
}
